Common/Core/TrackSelection.h

```diff
@@ -77,37 +78,54 @@ class TrackSelection
   template <typename T>
   bool IsSelected(T const& track, const TrackCuts& cut)
   {
+    bool isRun2 = track.trackType() == o2::aod::track::Run2Track || track.trackType() == o2::aod::track::Run2Tracklet;
```

Comment:
Why do you need to evaluate this on each track?

Reply:
You want to autodetect this setting?

Reply:
how can this can be implemented?